sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs

```diff
@@ -44,7 +44,7 @@ internal DevicesClient(RegistryManagerRestClient registryManagerClient, TwinRest
         /// Create or update a device identity.
         /// </summary>
         /// <param name="deviceIdentity">the device identity to create.</param>
-        /// <param name="precondition">The condition on which to perform this operation. To create a device identity, this value must be equal to <see cref="IfMatchPrecondition.Unconditional"/>.</param>
+        /// <param name="precondition">The condition on which to perform this operation. To create a device identity, this value must be equal to <see cref="IfMatchPrecondition.UnconditionalIfMatch"/>.</param>
```

Comment:
Shouldn't this be "IfMatchPrecondition.IfMatch" so that the SDK uses the null ETag value as the if-match header instead of "*"?